modules/cmpapi/test/CallResponder.test.ts

```diff
@@ -98,23 +98,27 @@ describe('CallResponder', (): void => {
   it('should use custom "getTCData" command handler for "addEventListener" and "removeEventListener" commands', (): void => {
 
     const customCommandCallback = sinon.stub();
-  
+
     const callResponder = new CallResponder({
       [TCFCommand.GET_TC_DATA]: (): void => {
+
         customCommandCallback();
+
       },
     });
-  
+
+    CmpApiModel.tcModel = TCModelFactory.withGVL();
```

Comment:
Looks like without a tcModel defined, the code path leads down this path:
![Screen Shot 2021-03-18 at 11 24 38 AM](https://user-images.githubusercontent.com/23621173/111677542-90649d80-87dc-11eb-8910-b31b6b67d83d.png)